platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/internal/cc/impl/DefaultConfigurationCacheIO.kt

```diff
@@ -131,6 +135,7 @@ class DefaultConfigurationCacheIO internal constructor(
             writeCollection(sideEffects) {
                 addressSerializer.write(this, it)
             }
+            write(buildInvocationScopeId.id.asString())
```

Comment:
💅 I'd make this the 1st field, it seems to deserve it

Reply:
First field it is